firefox-ios/Client/Frontend/Microsurvey/Survey/View/MicrosurveyTableViewCell.swift

```diff
@@ -35,28 +35,44 @@ class MicrosurveyTableViewCell: UITableViewCell, ReusableCell, ThemeApplicable {
     private lazy var radioButton: UIImageView = .build { imageView in
         imageView.contentMode = .scaleAspectFit
         imageView.image = UIImage(named: UX.Images.notSelected)
-        imageView.accessibilityIdentifier = AccessibilityIdentifiers.Microsurvey.Survey.radioButton
+        imageView.isAccessibilityElement = false
     }
 
     private lazy var optionLabel: UILabel = .build { label in
         label.adjustsFontForContentSizeCategory = true
         label.font = FXFontStyles.Regular.body.scaledFont()
         label.numberOfLines = 0
         label.lineBreakMode = .byWordWrapping
+        label.isAccessibilityElement = false
     }
 
     var checked = false {
         didSet {
             let checkedButton = UIImage(named: UX.Images.selected)
             let uncheckedButton = UIImage(named: UX.Images.notSelected)
             self.radioButton.image = checked ? checkedButton : uncheckedButton
+            accessibilityValue = optionA11yValue
         }
     }
 
+    var optionA11yValue: String {
+        let selectedLabel: String = .Microsurvey.Survey.SelectedRadioButtonAccessibilityLabel
+        let unselectedLabel: String = .Microsurvey.Survey.SelectedRadioButtonAccessibilityLabel
```

Comment:
This should probably be `UnselectedRadioButtonAccessibilityLabel` here 😉

Reply:
ah, nice catch! thank you!